shared/crypto/README.md

```diff
@@ -49,13 +49,11 @@ flowchart LR
     subgraph S[scrypt lib]
         B["key(salt, pass, ...)"]
     end
-    subgraph AES-GCM
+    subgraph SecretBox
```

Comment:
Above the flowchart, can you add a sentence saying "An example of a SecretBox is a cipher suite such as AES-GCM". It'll provide a concrete example for new readers.

Reply:
The actual cipher suite used by secretbox is XSalsa20+Poly1305 I will add something along the same lines